scripts/generate_fake_driver.sh

```diff
@@ -197,6 +197,11 @@ while IFS= read -r device_raw; do
         filename=${filename%.cfg}_2.cfg
     fi
 
+    # There are two "Razer Ornata V3"
+    if [ "$device_pid" = "02A1" ]; then
```

Comment:
I think if you do 028F here the _fake_driver diff should be cleaner. But doesn't matter much also